alibi_detect/cd/tensorflow/mmd.py

```diff
@@ -115,4 +115,8 @@ def score(self, x: Union[np.ndarray, list]) -> Tuple[float, float, np.ndarray]:
              for _ in range(self.n_permutations)]
         )
         p_val = (mmd2 <= mmd2_permuted).mean()
-        return p_val, mmd2, mmd2_permuted
+        # compute distance threshold
+        idx_threshold = int(self.p_val * len(mmd2_permuted))
+        distance_threshold = np.sort(mmd2_permuted)[::-1][idx_threshold]
```

Comment:
Not too bothered here but the torch version uses `torch.sort`, any reason why tensorflow doesn't use `tf.sort`?

Reply:
No good reason. The TF version worked fine with just copying across the legacy `np.sort` code from `predict`, whereas the torch version required updating. Agree though we should use the backend-specific versions for both!